source/main/utils/handleDiskSpace.js

```diff
@@ -181,7 +181,9 @@ export const handleDiskSpace = (
       logger.error('[DISK-SPACE-DEBUG] Unknown error', error);
       resetInterval(DISK_SPACE_CHECK_MEDIUM_INTERVAL);
     }
-    await getDiskSpaceStatusChannel.send(response, mainWindow.webContents);
```

Comment:
As far as I understand this sends the latest disk space information to the UI. If you will remove that `await` it might produce race conditions and unexpected behaviour. Please correct me if I'm wrong.

Reply:
@przemyslaw-wlodek i've forgotten to push the actual changes. This should get disk report and fire once to the ui:
```
    const diskReport = await getDiskCheckReport(stateDirectoryPath);
```